.gitignore

```diff
@@ -15,3 +15,4 @@ distribution/smarthome/runtime/etc/mapdb/storage.mapdb.t
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb.p
 bundles/storage/org.eclipse.smarthome.storage.mapdb.test/userdata/mapdb/storage.mapdb.t
+/.project
```

Comment:
please remove this again because we have .project files in the repo on purpose for the normal ESH IDE setup.